### ISSUE TITLE
dateTimeFormat.resolved.pattern needed

### ISSUE THREAD

**Issue by iegik** [andyearnshaw/Intl.js#129]
Sometimes need to get a date pattern for `input[type=date]` fields for example. Google have following solution:
```
(new Intl.DateTimeFormat()).resolved.pattern // "M/d/y" for "en-US" in Google Chrome
```

**Comment by caridy** [comment]
Why will you ever need access to that? Isn't the default date type matches the user's system configuration, or the page's definition? Just curious about it.

For that I remember, that is not part of the Ecma 402. We can make a case to expose those as part of the low level api, but it will take time.

**Comment by iegik** [comment]
Yes, it matches. This is the problem. `<input type="date">` will give You short format, but sometimes in design or other purposes, You want show to the end user long date or custom format.

**Comment by caridy** [comment]
Ok, fair enough. Let's start the discussion about adding this feature to the specification, and see how it goes.

**Comment by caridy** [comment]
I have open an issue to discuss this new feature as part of the ECMA 402 specification: https://github.com/tc39/ecma402/issues/21

I will keep this issue open until we get to a resolution.

**Comment by iegik** [comment]
thanks

**Comment by caridy** [comment]
Closing this since it depends on ecma402.
